capstone-frontend/src/components/AssignmentCard.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+
+import CommentCard from './CommentCard';
+
+import '../styles/Groups.css';
+
+
+class AssignmentCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: []
+    }
+  }
+  onComment = (e) => {
+    e.preventDefault();
+    let data = {
+      "assignmentID": this.props.assignment.id,
+      "text": e.target[0].value,
+      "userID": window.localStorage.getItem("userID"),
+      "whenCreated": new Date()
+    };
+    fetch("/api/comments", {method: "post", body: JSON.stringify(data)})
+        .then(this.fetchComments())
+        .catch(function(err) {
+          //TODO #61: Centralize error output
+          alert(err); 
+        });
+  }
+
+  fetchComments = () => {
+    fetch(`/api/comments?assignmentID=${this.props.assignment.id}`)
+        .then(response => response.json()).then(response => this.setState({comments: response}))
+        .catch(function(err) {
+          //TODO #61: Centralize error output
+          alert(err); 
+        });
+  }
+
+  componentDidMount() {
+    this.fetchComments();
+  }
+
+  render() {
+    return (
+      <div className="assignment-border">
+        <div> {this.props.assignment.text} </div>
+        {this.state.comments.map(c => <CommentCard key={c.id} comment={c} />)}
+        <form id="comment-form" onSubmit={this.onComment}>
+          <div>
+            <label htmlFor="text"> Comment: </label>
+          </div>
+          <div>
+            <textarea rows="3" cols="75" type="text" id="text" name="text" />
```

Comment:
Maybe the default comment box should be a little shorter? It seems to take up a lot of space in your demo image, if the user wants more space then they have the option to expand it

Reply:
Good idea. I'll change it to have `rows="1"` by default.